config.json

```diff
@@ -288,296 +307,395 @@
+        "uuid": "818c6472-b734-4ff4-8016-ce540141faec",
+        "practices": ["list-methods"],
+        "prerequisites": ["conditionals", "lists", "list-methods", "loops"],
+        "difficulty": 1
+      },
+      {
+        "slug": "acronym",
+        "name": "Acronym",
+        "uuid": "038c7f7f-02f6-496f-9e16-9372621cc4cd",
+        "practices": ["string-methods"],
+        "prerequisites": ["basics", "loops", "strings", "string-methods"],
+        "difficulty": 2
+      },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "cb581e2c-66ab-4221-9884-44bacb7c4ebe",
+        "practices": ["comparisons"],
+        "prerequisites": ["basics", "numbers", "comparisons", "conditionals"],
+        "difficulty": 2
+      },
+      {
+        "slug": "pig-latin",
+        "name": "Pig Latin",
+        "uuid": "a5bc16da-8d55-4840-9523-686aebbaaa7e",
+        "practices": ["conditionals"],
+        "prerequisites": ["basics", "bools", "conditionals"],
         "difficulty": 2
       },
       {
         "slug": "protein-translation",
         "name": "Protein Translation",
         "uuid": "c89243f3-703e-4fe0-8e43-f200eedf2825",
-        "practices": ["dicts", "dict-methods", "loops"],
+        "practices": ["loops"],
         "prerequisites": [
           "basics",
           "conditionals",
-          "dicts",
           "lists",
           "loops",
           "strings",
           "string-methods"
         ],
-        "difficulty": 3
+        "difficulty": 2
       },
       {
-        "slug": "armstrong-numbers",
-        "name": "Armstrong Numbers",
-        "uuid": "d9ceb246-b518-42b9-9fa3-112e25c7ecd8",
-        "practices": ["generator-expressions"],
+        "slug": "scrabble-score",
+        "name": "Scrabble Score",
+        "uuid": "d081446b-f26b-41a2-ab7f-dd7f6736ecfe",
+        "practices": ["dicts"],
         "prerequisites": [
           "basics",
           "lists",
           "loops",
-          "numbers",
+          "dicts",
           "strings",
           "string-methods"
         ],
-        "difficulty": 1
+        "difficulty": 2
       },
       {
-        "slug": "perfect-numbers",
-        "name": "Perfect Numbers",
-        "uuid": "c23ae7a3-3095-4608-8720-ee9ce8938f26",
+        "slug": "atbash-cipher",
+        "name": "Atbash Cipher",
+        "uuid": "02b91a90-244d-479e-a039-0e1d328c0be9",
+        "practices": ["string-methods"],
+        "prerequisites": [
+          "basics",
+          "conditionals",
+          "loops",
+          "lists",
+          "list-methods",
+          "strings",
+          "string-methods"
+        ],
+        "difficulty": 2
+      },
+      {
+        "slug": "word-count",
+        "name": "Word Count",
+        "uuid": "04316811-0bc3-4377-8ff5-5a300ba41d61",
+        "practices": ["dicts"],
+        "prerequisites": [
+          "basics",
+          "loops",
+          "strings",
+          "string-methods",
+          "dicts"
+        ],
```

Comment:
But that is an _example_ (proof the problem is solvable) not an exemplar.  This problem is both solvable without regex, and in some cases faster (and also good for practicing `string-methods`).  It is also a really nice setup for Counter - which is an extended `dict`.  

But I agree that it is on the line for practicing `dicts`.  Again, this is a situation where if we remove this, we are left with `etl`, `space-age`, `scrabble-score` and .... `fourth` (which at L5 is quite a jump).  This feels like another "hole" that needs additional practice exercises....

Reply:
Changed the practices.  Left `regular-expressions` off the pre-requs here, so that the exercise can remain unlocked since the concept is TBD right now.